### PR TITLE
[CU-86b1pckzr] Add run events command to CLI

### PR DESCRIPTION
**Pull request by eric-j22**

### CODE REVIEW

**Comment by platform-automation-dnastack** [comment]
Task linked: [CU-86b1pckzr Add events command to run cli](https://app.clickup.com/t/86b1pckzr)

File: dnastack/client/workbench/ewes/models.py [hunk 1]
```diff
@@ -111,6 +131,7 @@ class ExtendedRun(BaseModel):
     task_logs: Optional[List[Log]]
     task_logs_url: Optional[str]
     outputs: Optional[Dict]
+    events: Optional[List[RunEvent]]
```
Comment (eric-j22) [inline dnastack/client/workbench/ewes/models.py:134]: This causes events to be also listed when listing runs, is that desired behaviour or should it be changed?

**Review by tianning-zhang — APPROVED** [review]
LGTM

File: dnastack/client/workbench/ewes/models.py (hunk 1 above)
Comment (patmagee) [reply dnastack/client/workbench/ewes/models.py:134]: That is a bit annoying.... You could always set the value to `None` in the `describe` / `list` endpoint for runs?

Comment (eric-j22) [reply dnastack/client/workbench/ewes/models.py:134]: Yeah that makes sense I can do that

Comment (eric-j22) [reply dnastack/client/workbench/ewes/models.py:134]: Nevermind, I think doing this actually breaks a few tests/some functionality. I could also just create a different class specifically for events that will only be populated by events.

Comment (patmagee) [reply dnastack/client/workbench/ewes/models.py:134]: Sounds good!

**Pull request merged**